service/src/main/java/bio/terra/cbas/service/RunSetsService.java

```diff
@@ -196,7 +206,7 @@ public void triggerWorkflowSubmission(
     }
 
     // For each Record ID, build workflow inputs and submit the workflow to Cromwell
-    List<RunStateResponse> runStateResponseList =
+    ImmutablePair<List<RunStateResponse>, List<String>> runStateResponse =
```

Comment:
Nit: This return value is getting towards a "wrap it in a `record` type" level of complexity